index.html

```diff
@@ -1,19 +1,35 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Web-design</title>
-    <link rel="stylesheet" href="css/sheet.css">
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Web-design</title>
+  <link rel="stylesheet" href="css/sheet.css">
 </head>
 <body>
-   <header >
-       <nav>
-            <div class="bg-black"><img src="image/LOGO_RW.png"></div>
-       </nav>
-      
-   </header>
+  <header>
+    <nav>
+      <div class="bg-black"><img src="image/LOGO_RW.png"></div>
+    </nav>
+    <div>
+    </div>
+  </header>
+  <section>
+    <div class="container">
+     <div class="text-white">
+      <h1>
+        Land of 1000 Hills
+      </h1>
+      <div>
+        Rwanda is called the land of thousands hills and mountains because it is splending scenic little country
+      </div>
+     </div>
+      <div class="rwanda">
```

Comment:
replace `.rwanda` with `.col-50`. and inside .col-50, add a div with class .image. this will help you to style image while making sure that you .col-50 as a container